SConstruct

```diff
@@ -0,0 +1,4 @@
+# -*- python -*-
```

Comment:
This is probably just my ignorance but why do we need SContruct files for a python only program?  Do we need this fore eups somehow?